src/main/python/deep_qa/layers/entailment_models/word_alignment.py

```diff
@@ -55,6 +55,7 @@ def _align(source_embedding, target_embedding, source_mask, target_mask, normali
                 # (batch_size, source_length, target_length)
                 s2t_mask = K.expand_dims(float_source_mask, dim=-1) * K.expand_dims(float_target_mask, dim=1)
                 flattened_s2t_mask = last_dim_flatten(s2t_mask)
+                # pylint: disable=redefined-variable-type
```

Comment:
this is possible, but it's not transparent to me which pylint is used since it uses a cached version...